### DISCUSSION TITLE
Theia build-in logos

### DISCUSSION THREAD

**Discussion by wassimkha** [eclipse-theia/theia#10306] [Q&A]
Hi,

I opened an issue two weeks ago about the SVGs not working due to the new update and that's been fixed. Here's that post: https://github.com/eclipse-theia/theia/discussions/10223

Although, there's one last issue that I described here, but got no response, so just curious if there's an issue for this, or is it something that I need to modify locally? Here's that comment left: https://github.com/eclipse-theia/theia/discussions/10223#discussioncomment-1438648

any help is appreciated!

**Comment by vince-fugnitto** [comment]
@wassimkha thank you for the discussion and sorry about that, probably got lost in all the conversations that was happening in the original issue. Can you confirm if executing the command `reset workbench layout` resolves the problem? If not, is there any way we can test with your extension, or provide a way to test?
